src/wagtailtrans/models.py

```diff
@@ -110,7 +110,7 @@ def __str__(self):
         return force_text(dict(settings.LANGUAGES).get(self.code))
 
     def has_pages_in_site(self, site):
-        return self.pages.filter(path__startswith=site.root_page.path).exists()
+        return TranslatablePageItem.objects.filter(page__path__startswith=site.root_page.path).exists()
```

Comment:
https://github.com/wagtail/wagtail/blob/master/wagtail/core/query.py#L356

Reply:
Maybe we can use this, since it's more in line with the wagtail approach?

Reply:
nevermind, thought it was a page model..

Reply:
Should be rewritten to `has_translatable_page_items_in_site` though, more explicit